src/codecs/fsk/fsk.h

```diff
@@ -38,6 +38,18 @@ typedef struct _fsk_encoder_api {
      */
     uint32_t (*get_symbol_delay)(fsk_encoder *encoder);
 
+    /**
+     * @param encoder
+     * @return Pointer to packet data
+     */
+    uint8_t *(*get_data)(fsk_encoder *encoder);
```

Comment:
Exposing the data here isn't really the right thing to do for this API struct, but since this "fsk encoder" is now the only encoder abstraction we have, let's go with this. :)